### ISSUE TITLE
Possible to make cutscenes skippable?

### ISSUE THREAD

**Issue by itismeDonutello** [Lyall/RF5Fix#6]
First of all thank you so much, works great, you are doing god's work! I'm one of the people with the 'black screen instead of cutscenes' bug, so I can't progress in the game, would it be possible to add an option for skipping the cutscenes altogether?

**Comment by Lyall** [comment]
By cutscenes are you referring to the anime-style pre-rendered movies or actual in-game cutscenes?

**Comment by itismeDonutello** [comment]
I meant the anime per-rendered movies (eg. intro movie, character introductions), sorry I wasn't more clear. Basically some people get stuck on them, black screen appears and the game is non-responsive. I can't progress further than day 1, because I'm stuck on the first character intro movie right after you wake up.

**Comment by Lyall** [comment]
I could hook in to the video player and make it skip/not even try and load the videos. Though I can't test to make sure this fixes that bug as I haven't experienced it myself. So I'd need your help to test it. If you'd be happy to test out a build and give me feedback then I can work on this as an option.

**Comment by itismeDonutello** [comment]
Damn that would be amazing, I will very gladly test it and give you feedback. Gotta say tho I don't know anything about programming, so I'm limited in my testing skills lel

**Comment by Lyall** [comment]
All you need to do is just extract the builds I post in this issue and test them out. No programming knowledge needed. Are you able to describe how far you actually get? Like do you see the main title screen with/without the mod? Can you get in-game?

**Comment by itismeDonutello** [comment]
Yes, I can get in-game no problem, the first game intro before the main menu doesn't play either, but I can skip it by left clicking mouse on it. Your mod also works perfect since it just skips the intro/logos. The problem arises when the first character is introduced  with a cutscene (Priscilla, I think) right after you wake up (after the farm tutorial). So she greets you, then MC says "Mnnn" and then black screen appears instead of per-rendered Priscilla video introduction. I can still see and move the mouse, but the screen is black, if I click anything nothing happens. Basically something prevents these videos from being played, I think just adding the option for skipping them altogether should probably solve the issue. Let me know if you need more info

**Comment by Lyall** [comment]
Alright, I think that should be enough info. So this build I'm attaching has should skip videos, it basically stops a video before it even starts so let's give that a try and see how it does.

[RF5Fix_v0.0.9_VideoBugv1.zip](https://github.com/Lyall/RF5Fix/files/9161258/RF5Fix_v0.0.9_VideoBugv1.zip)

**Comment by itismeDonutello** [comment]
Damn sorry, doesn't work for some reason. Basically still have black screen, mouse icon moving, nothing happens when i click anything. It does multiply my mouse icon now tho when i move it haha
It's weird because with your mod it skips intro before the main menu no problem, I don't have to click anything at all, kinds thought it would work

**Comment by Lyall** [comment]
Alright, makes sense that this will be a bit difficult. I'm basically working in the dark without being able to replicate the issue on my end. Here's a new build to try.

[RF5Fix_v0.0.9_VideoBugv2.zip](https://github.com/Lyall/RF5Fix/files/9168820/RF5Fix_v0.0.9_VideoBugv2.zip)

**Comment by itismeDonutello** [comment]
Understandable! Please, no pressure, I already appreciate you so much trying to fix it. 
I tested the new build, the mouse icon replication doesn't happen anymore, but other than that there is no change (black screen, mouse moving, nothing else is happening).

**Comment by Lyall** [comment]
So unfortunately on this I think I'm out of ideas. Ideally something to try next would be to patch out cutscene triggers but that's outside of the scope of a BepInEx mod. I think realistically here the best hope for this issue is for the devs to patch it.

**Comment by itismeDonutello** [comment]
Alright, thanks a lot for trying anyway! Hopefully they'll patch it soon, it's pretty disappointing that they haven't released any patches yet, as there seem to be plenty of bugs around. but oh well

**Comment by Spike2D** [comment]
There is an addon on Nexus to skip dialogue and some cutscenes, might be a good idea to use that or learn from it.
